src/stores/SetupStore.js

```diff
@@ -147,19 +151,23 @@ var SetupStore = assign(Object.create(EventEmitter.prototype), {
     var isoversion = machine.isoversion();
     var required = {};
     var vboxfile = path.join(util.supportDir(), virtualBox.filename());
-    var vboxNeedsInstall = !virtualBox.installed();
+    var vboxNeedsInstall = !util.isLinux() && !virtualBox.installed();
```

Comment:
Instead of all the changes in `requiredSteps`, we could just add the following code at the start of the `run` function:

```

if (util.isLinux()) {
  yield 'localhost';
}
```